.github/workflows/main.yml

```diff
@@ -79,6 +80,34 @@ jobs:
           path: ./*
           key: ${{ github.ref }}-${{ github.sha }}-build-release
 
+  build-docs:
+    needs: setup
+    runs-on: ubuntu-latest
+    steps:
+      - name: Restore setup
+        uses: actions/cache@v3
+        with:
+          path: ./*
+          key: ${{ github.ref }}-${{ github.sha }}-setup
+
+      - name: Set up Ruby
+        uses: ruby/setup-ruby@v1
+        with:
+          ruby-version: ${{ env.RUBY_VERSION }}
+          bundler-cache: true
+          cache-version: 1
```

Comment:
If I get this right then this value will have to be bumped manually according to the docs:

> In some rare scenarios (like using gems with C extensions whose functionality depends on libraries found on the system at the time of the gem's build) it may be necessary to ignore contents of the cache and get and build all the gems anew. In order to achieve this, set the `cache-version` option to any value other than `0` (or change it to a new unique value if you have already used it before.)

From what I am reading here I believe we can leave this empty for now.
```suggestion
```

Reply:
Ah yes, I thought it might be needed so I added it. I'll push a change to remove that line.